backend/app/ai/routers/search.py

```
@@ -1,5 +1,6 @@
-from fastapi import APIRouter, Depends, HTTPException, Query
+from fastapi import APIRouter, Depends, Query
 from sqlalchemy.orm import Session
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)